packages/dev/core/src/node.ts

```diff
@@ -814,6 +814,38 @@ export class Node implements IBehaviorAware<Node> {
         return this._ranges[name] || null;
     }
 
+    /**
+     * Clone the current node
+     * @param name Name of the new clone
+     * @param newParent New parent for the clone
+     * @param doNotCloneChildren Do not clone children hierarchy
+     * @returns the new transform node
+     */
+    public clone(name: string, newParent: Nullable<Node>, doNotCloneChildren?: boolean): Nullable<Node> {

```

Comment:
I think we can have the same default values than in the `Mesh.clone` method?

```typescript
public clone(name: string = "", newParent: Nullable<Node> = null, doNotCloneChildren?: boolean): Nullable<Node>
```

Also, why not returning `Node` instead of `Nullable<Node>`? The cloned node can't be `null`.

Reply:
Clone in TransformNode returns Nullable, so I just copied the same method. But I think there's no problem in taking Nullable out.

Reply:
Scratch that, we get some typing problems if clone doesn't return a Nullable. Probably related to the node hierarchy.

Reply:
It's `AbstractMesh` which returns null because it does not implement the method... I don't think we can go around it, unfortunately, so we must return a `nullable` indeed.